x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/enrollment/ExternalEnrollmentTokenGeneratorTests.java

```diff
@@ -85,8 +85,9 @@ public void testCreateSuccess() throws Exception {
         final CommandLineHttpClient client = mock(CommandLineHttpClient.class);
         when(client.getDefaultURL()).thenReturn("http://localhost:9200");
```

Comment:
Maybe this shouldn't be mocked, or be mocked with a different value.

Reply:
True, leftover , thanks !